tests/test_main.py

```diff
@@ -561,5 +569,25 @@ def test_resume_no(monkeypatch):
             skip_row_id=False,
             output_delimiter=None,
             compression=False,
-            field_size_limit=None
+            field_size_limit=None,
+            verify_ssl=True
         )
+
+
+@pytest.mark.parametrize('ssl_argvs, expected_value', [
+    ([], True),
+    (['--no_verify_ssl'], False),
+    (['--ca_bundle', '/path/to/cert'], '/path/to/cert'),
+    (['--ca_bundle', '/path/to/cert', '--no_verify_ssl'], False)
```

Comment:
Could you not parametrize this to make it less cryptic?

Also, this detail should be explained in the docs.

Reply:
> Could you not parametrize this to make it less cryptic?

Do you think this so cryptic that it'd justify the copy pasting of test functions? It's true that `pytest.mark.parametrize` doesn't help readability usually, but I think I still prefer the parametrized version.

> Also, this detail should be explained in the docs.

Do users need an explanation that saying us "here's a certificate for verification, oh also don't verify anything" will result in skipping SSL verification? I think this is a pretty intuitive behavior.

Reply:
`Using this flag will cause the argument for ca_bundle to be ignored` <- not that hard.

Also, copying and pasting of test functions is not a problem to me. Not being able to understand what situation a test is covering is a problem to me.